MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -2329,22 +2329,26 @@ private string GetFixedSizeScalingFilter(Video3DFormat? threedFormat, int reques
         /// <summary>
         /// Gets the number of threads.
         /// </summary>
-        public int GetNumberOfThreads(EncodingJobInfo state, EncodingOptions encodingOptions, string outputVideoCodec)
+        public static int GetNumberOfThreads(EncodingJobInfo state, EncodingOptions encodingOptions, string outputVideoCodec)
         {
-            if (string.Equals(outputVideoCodec, "libvpx", StringComparison.OrdinalIgnoreCase))
+            if (outputVideoCodec != null && string.Equals(outputVideoCodec, "libvpx", StringComparison.OrdinalIgnoreCase))
```

Comment:
According to the docs, you don't need ```outputVideoCodec != null``` 

```
Parameters

a
    String

The first string to compare, or null.

b
    String

The second string to compare, or null.
```

https://docs.microsoft.com/en-us/dotnet/api/system.string.equals?view=net-5.0#System_String_Equals_System_String_System_String_

Reply:
@BaronGreenback Oh true, that's some testing bits I left behind.